src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -20,6 +20,19 @@ internal static partial class SocketPal
         // PlatformNotSupportedException instead.
         public const int ProtocolInformationSize = 0;
 
+        public readonly static bool SupportsMultipleConnectAttempts = GetPlatformSupportsMultipleConnectAttempts();
+        private readonly static bool SupportsDualModeIPv4PacketInfo = GetPlatformSupportsDualModeIPv4PacketInfo();
```

Comment:
Nit: s_supportsMultipleConnectAttempts and s_supportsDualModeIPv4PacketInfo (unless there's some strong reason we went with these named this way)